src/plugins/Mirror.cpp

```diff
@@ -153,18 +154,58 @@ void Mirror::readXML(XMLfileUnits& xmlconfig)
 	/** Meland2004 */
 	if(MT_MELAND_2004 == _type)
 	{
-		_melandParams.use_probability_factor = true;
-		_melandParams.velo_target = 0.4;
```

Comment:
don't delete this default value!
currently, if meland/velo_target is not specified in the xml, this value is undefined and can be anything.
It would be good practice to initialize the _melandParams with some meaningful values inside of the constructor, or using inline variable initialization (inside of the header file, by adding the default value in curly brackets like this):
```c++
	struct MelandParams {
		bool use_probability_factor {false};
		double velo_target {0.4};
		float fixed_probability_factor {-1};
	} _melandParams;
```

Reply:
Ok. I added the default values during initializiation in the header-file as suggested.
But if meland/velo_target is not specified in the xml, wouldn't the whole program be aborted?

Reply:
As you mentioned, leaving the variables uninitialized seems like its not good practice, even though they are overwritten afterwards. Thanks for the hint!

Reply:
true, it would be aborted, but it's better anyways, especially in case this plugin is used in unit-tests, etc.